### ISSUE TITLE
D and G loss value for good tuned CycleGAN? 

### ISSUE THREAD

**Issue by kpagels** [junyanz/pytorch-CycleGAN-and-pix2pix#806]
You guys obviously did a lot of training. 

What values did you experience for the discriminator and generator loss when training your networks? Do you have some loss plots? 

Does some of them converge to some value?

And when did you know that the system collapsed? Was that only determined by the translated images?

**Comment by junyanz** [comment]
1. We haven't saved our training plots. But if you train a model with default datasets and parameters,  you should be able to see a typical cyclegan training loss curve. 

2. The GAN losses we used do not converge as it's minimax optimization problem. There are other types of GAN losses that converge (e.g., WGAN family)

3. If D's loss is almost 0 for a while, your training might have collapsed. You can inspect the visual results directly.

**Comment by kpagels** [comment]
Thanks!
